pkg/build/controller/build/pluginconfig/config_test.go

```diff
@@ -1,80 +0,0 @@
-package pluginconfig
-
-import (
-	"reflect"
-	"testing"
-
-	testtypes "github.com/openshift/origin/pkg/build/controller/build/pluginconfig/testing"
-	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
-	"github.com/openshift/origin/pkg/cmd/server/apis/config/latest"
-	configapiv1 "github.com/openshift/origin/pkg/cmd/server/apis/config/v1"
-
-	// install server api
-	_ "github.com/openshift/origin/pkg/api/install"
-	"github.com/openshift/origin/pkg/api/legacy"
-	_ "github.com/openshift/origin/pkg/cmd/server/apis/config/install"
-)
-
-func TestGetPluginConfig(t *testing.T) {
```

Comment:
@deads2k where did this moved?

Reply:
> @deads2k where did this moved?

This test is broken and the code also needs a bit of love since it is only intended to manage consuming a single type.  I promise to come back to it.